CHANGELOG.md

```diff
@@ -4,9 +4,10 @@
 - Don't trigger GitHub 'Run test' workflow on a push that only changes README.md and/or CHANGELOG.md
 - Avoid triggering duplicate GitHub workflows. When pushing to a branch that's in a PR, only trigger tests on the push not on the PR sync also.
 ### Added
-- Add geothermal heat pump (`GHP`), also known as ground-source heat pump (GSHP), to the REopt model for serving heating and cooling loads
-    - The unregistered `GhpGhx.jl` package (https://github.com/NREL/GhpGhx.jl) is now a dependency of REopt.jl, and this package sizes the ground heat exchanger (GHE) and gets the hourly electric consumption of the `GHP` for the specified heating and cooling loads that it serves.
-    - The `GHP` size and dispatch of the different `GHP` options is pre-determined by the `GhpGhx.jl` package, so the REopt model just chooses one or none of the GHP options with a binary decision variable.
+- Add geothermal heat pump (`GHP`), also known as ground-source heat pump (GSHP), to the REopt model for serving heating and cooling loads (typically the benefits include electrifying the heating load and improving the efficiency of cooling).
+    - The unregistered `GhpGhx` package (https://github.com/NREL/GhpGhx.jl) is a "conditional" dependency of REopt by using the Requires.jl package, and this package sizes the ground heat exchanger (GHE) and gets the hourly electric consumption of the `GHP` for the specified heating and cooling loads that it serves.
+    - The `GhpGhx` module calls for sizing the GHE can only be done if you first "add https://github.com/NREL/GhpGhx.jl" to the environment and then load the package by "using GhxGhx" before running REopt with `GHP`.
```

Comment:
Also, maybe say:

first add https://github.com/NREL/GhpGhx.jl" to the environment **from the package manager** and then...